neural_compressor/torch/quantization/config.py

```diff
@@ -995,6 +997,7 @@ def __init__(
         self.act_granularity = act_granularity
         self.act_algo = act_algo
         self.excluded_precisions = excluded_precisions
+        self.device = device
```

Comment:
I have a general question: should we infer the `device` from the model, or should we ask users to set it explicitly? 
If users have already placed the model on XPU using sth like `model.to(xpu)`, I believe we should infer it from the model by checking the device information of the model's parameters.